project/config/hseni/config/user.role.supervisor_user.yml

```diff
@@ -22,6 +22,11 @@ dependencies:
     - node.type.profile
     - node.type.publication
     - node.type.webform
+    - taxonomy.vocabulary.link_types
```

Comment:
I don't think we want users to add any taxonomy apart from site topics.

Reply:
Thanks, done